tap_asana/streams/subtasks.py

```diff
@@ -59,13 +60,26 @@ def get_objects(self):
         opt_fields = ",".join(self.fields)
         bookmark = self.get_bookmark()
         session_bookmark = bookmark
+        modified_since = bookmark.strftime("%Y-%m-%dT%H:%M:%S.%f")
         for workspace in self.call_api("workspaces"):
             for project in self.call_api("projects", workspace=workspace["gid"]):
                 project_ids.append(project["gid"])
 
+        projects_total = len(project_ids)
+        projects_fraction = projects_total // 100 # near 1% of total projects
+
+        # iterate over all project ids and continue fetching
+        LOGGER.info("Fetching subtasks...")
         for indx, project_id in enumerate(project_ids, 1):
-            LOGGER.info("Fetching Subtasks for project: %s/%s", indx, len(project_ids))
-            tasks_list = self.call_api("tasks", project=project_id, opt_fields=opt_fields)
+            if (indx % projects_fraction == 0):
```

Comment:
it's done in order to get less logs. As in case of thousands of projects there're to many logs